docs/readthedocs/source/doc/Nano/QuickStart/pytorch_train_quickstart.md

```diff
@@ -66,7 +66,7 @@ class LitResnet(LightningModule):
     def __init__(self, learning_rate=0.05, num_processes=1):
         super().__init__()
 
-        self.save_hyperparameters()
+        self.save_hyperparameters('learning_rate', 'num_processes')
```

Comment:
https://pytorch-lightning.readthedocs.io/en/stable/common/hyperparameters.html#lightningmodule-hyperparameters these 2 calls seems to be equivalent?

Reply:
If not specifying the key arguments to the `save_hyperparameters()` function, python compiler seems unable to recognize the argument for the class (will be used in training), which leads to runtime error during my testing.